src/lightning_app/cli/lightning_cli.py

```diff
@@ -49,10 +51,89 @@ def main():
 @click.group()
 @click.version_option(ver)
 def _main():
-    register_all_external_components()
+    # register_all_external_components()
     pass
 
 
+@_main.group()
+def show():
+    """Show given resource."""
+    pass
+
+
+@show.command()
```

Comment:
This doesn't work locally. So we need a mechanism to inform the users.

Reply:
@tchaton  Actually, now I think about it. Maybe we could unify the behaviour of local and cloud app running? It would take some time but we have (almost) all peaces of the puzzle.
We could locally stream all logs to some output file (e.g. somewhere in `.lightning` directory) and read them with `show logs`.

So by default running locally would actually return control right after starting an app instead of hanging and printing output. 

```
lightning run app app.py [--cloud] 

lightning show logs --follow.  # print logs as they go -> we have no difference between behaviour locally and on cloud
```

What do you think? Anywho - something to think about :D

Reply:
Yes, why not? Personally, I think it is a bit of overkill in the grand scheme of all the things to build, but I can imagine some users using this feature to isolate logs from a machine.